.travis.yml

```diff
@@ -10,7 +10,9 @@ before_install:
 
 # default script for jobs, that do not have any specified
 script:
-  - ./scripts/buildChanged.sh ${DIR:=.} "${JDK:=adopt@~1.8.202-08}" "${PRE_CMD:=echo NOOP}" "${CMD:=+$DIR/testChanged}"
+  - |
+    [ "$TRAVIS_EVENT_TYPE" == "cron" ] && JDK="adopt@~1.11.0-2" || JDK="adopt@~1.8.202-08" 
```

Comment:
What does the `|` do?

Reply:
It switches to YAML template mode. Otherwise yaml parser is not happy about `[` and `]` characters.